docs/releasing.md

```diff
@@ -0,0 +1,28 @@
+# Releasing process
+
+To create a new release there are some steps is required:
+
+## 1. Update Charts file
```

Comment:
I believe the first thing is having a proper tag, then creating an image and then updating the chart to use that image?

Reply:
The first thing we need to do is updating the chart with a new tag, when we're drafting a new release we should create a new tag the same as the one in the charts. we don't need to create an image because it's already handled and its build and push jobs are triggered on publish.